src/components/ContentBox/index.tsx

```diff
@@ -8,13 +8,14 @@ import { ContentBoxProps } from './types'
 
 export function ContentBox ({
   url,
+  width,
   title,
   children,
   imageSrc,
   subtitle
 }: ContentBoxProps) {
   return (
-    <Link href={url} isExternal>
+    <Link href={url} width={width} isExternal>
```

Comment:
Fixed a layout issue where the box link wasn't filling the full width of a page on mobile devices